std/stdio.d

```diff
@@ -4284,6 +4293,7 @@ __gshared
             .copy(                      // copy output of .sort to an OutputRange
                 stdout.lockingTextWriter()); // the OutputRange
         }
+        // turn into doc alone?
```

Comment:
Why is this a test?  It won't do anything.  Anyone against making this just documentation?